components/gitpod-cli/cmd/sync-await.go

```diff
@@ -25,6 +25,7 @@ var awaitSyncCmd = &cobra.Command{
 		id := hex.EncodeToString(h.Sum(nil))
 		lockFile := fmt.Sprintf("/tmp/gp-%s.done", id)
 
+		fmt.Printf("Awaiting '%s'... ", args[0])
```

Comment:
perhaps we can also make it say `Awaiting port ${port}` instead of `Awaiting ${port}` to make the output's purpose and source clear.

Reply:
`sync-await` is not about ports?

Reply:
Might be useful to introduce an explicit `--verbose` flag instead of just printing output. That would be very much in line with the Unix philosophy on CLIs:
> The [rule of silence](https://www.linfo.org/rule_of_silence.html#:~:text=The%20rule%20of%20silence%2C%20also,say%2C%20it%20should%20say%20nothing.), also referred to as the silence is golden rule, is an important part of the Unix philosophy that states that when a [program](https://www.linfo.org/program.html) has nothing surprising, interesting or useful to say, it should say nothing.